sshuttle/ssnet.py

```diff
@@ -243,7 +243,9 @@ def fill(self):
     def copy_to(self, outwrap):
         if self.buf and self.buf[0]:
             wrote = outwrap.write(self.buf[0])
-            if wrote > 0:
+            if wrote is None:
+                log("ssnet.py copy_to 'wrote' is None\n")
+            elif wrote > 0:
                 self.total_wrote += wrote
             self.buf[0] = self.buf[0][wrote:]
```

Comment:
if `wrote` is None, this selft.buf will be None as well? and the loop after might crash?

Reply:
@di1214 all good now?